src/main/java/com/aws/greengrass/clientdevices/auth/certificate/CertificateHelper.java

```diff
@@ -253,4 +269,97 @@ public static X500Name getX500Name(String commonName) {
 
         return nameBuilder.build();
     }
+
```

Comment:
All this chunk is copy pasta from Nucleus, I just tweaked it a bit to not have to select a provider, we just wanted the methods here.